src/Tickets/Commerce/Gateways/Square/Settings.php

```diff
@@ -77,13 +77,29 @@ class Settings extends Abstract_Settings {
 	public function get_settings(): array {
 		$is_connected = tribe( Merchant::class )->is_connected();
 
+		// Add the fee message similar to Stripe.
+		$plus_link_faq = sprintf(
+			'<a href="https://evnt.is/1b3u" target="_blank" rel="noopener noreferrer">%s</a>',
+			esc_html__( 'Learn more', 'event-tickets' )
+		);
+
+		$square_message = sprintf(
+			// Translators: %1$s: The Event Tickets Plus link.
```

Comment:
Might be good to add? Context.
```suggestion
			// Translators: %1$s: The Event Tickets Plus link - reads "Learn More".
```